app/src/main/java/com/quran/labs/androidquran/ui/PagerActivity.java

```diff
@@ -1078,7 +1078,6 @@ private void updateActionBarTitle(int page) {
     String sura = QuranInfo.getSuraNameFromPage(this, page, true);
     ActionBar actionBar = getSupportActionBar();
     if (actionBar != null) {
-      translationsSpinner.setVisibility(View.GONE);
```

Comment:
There must be a reason why we are hiding translation spinner here (in fact, there is a separate method for updating actionbar spinner).